cli/upgrade.rs

```diff
@@ -36,6 +36,8 @@ const ARCHIVE_NAME: &str = "deno-x86_64-pc-windows-msvc.zip";
 const ARCHIVE_NAME: &str = "deno-x86_64-apple-darwin.zip";
 #[cfg(target_os = "linux")]
 const ARCHIVE_NAME: &str = "deno-x86_64-unknown-linux-gnu.zip";
+#[cfg(target_os = "freebsd")]
+const ARCHIVE_NAME: &str = "deno-x86_64-unknown-freebsd.zip";
```

Comment:
```suggestion
const ARCHIVE_NAME: &str = "deno-x86_64-unknown-freebsd.zip";
```

this `ARCHIVE_NAME` is the target zip file name used in https://github.com/denoland/deno/releases
so we have to create a binary for freebsd

Reply:
I'm not sure how easy it'll be for you. We have a port in our infrastructure if that helps you: https://github.com/freebsd/freebsd-ports/tree/master/www/deno

Reply:
We need a FreeBSD build in CI to support it in upgrade. Adding this will just introduce errors.